backend/learnings/views.py

```diff
@@ -247,13 +247,12 @@ class GetUserActivityLog(APIView):
     authentication_classes = [TokenAuthentication]
 
     def get(self, request, page, username):
-        user = EUser.objects.get(username=username)
         if page == "dashboard":
-            activities = UserActivityLog.objects.filter(user_id=user).order_by("-created_at")
+            activities = UserActivityLog.objects.all().order_by("-created_at")
```

Comment:
this will display all user activities, including the activities of the users that the current users is not following.
display only these activities
- current user
- users that the current user is following
do not display this
- users that the current user is not following

Reply:
done po sir